src/test/java/uk/gov/justice/hmpps/prison/util/builders/OffenderBookingBuilder.kt

```diff
@@ -10,13 +10,13 @@ import uk.gov.justice.hmpps.prison.api.model.RequestForNewBooking
 import uk.gov.justice.hmpps.prison.service.DataLoaderRepository
 import uk.gov.justice.hmpps.prison.util.JwtAuthenticationHelper
 import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
 
 class OffenderBookingBuilder(
   var prisonId: String = "MDI",
-  var bookingInTime: LocalDateTime = LocalDateTime.now(),
+  var bookingInTime: LocalDateTime = LocalDateTime.now().minusDays(1),
   var fromLocationId: String = "OUT",
```

Comment:
since we will now releases some of the prisoners make sure their booking time is way in the past